spec/Entity/CreditMemoSpec.php

```diff
@@ -37,11 +38,16 @@ function let(ChannelInterface $channel, OrderInterface $order): void
         );
     }
 
-    function it_implements_credit_memo_interface(): void
+    function it_implements_a_credit_memo_interface(): void
     {
         $this->shouldImplement(CreditMemoInterface::class);
     }
 
+    function it_implements_a_resource_interface(): void
+    {
+        $this->shouldImplement(ResourceInterface::class);
```

Comment:
Do we need to check it? `CreditMemoInterface` extends `ResourceInterface`... I'm just not sure does it give us any value 🤔